test/test_aes.py

```diff
@@ -40,6 +53,33 @@ def test_cbc_encrypt(self):
             encrypted,
             b"\x97\x92+\xe5\x0b\xc3\x18\x91ky9m&\xb3\xb5@\xe6'\xc2\x96.\xc8u\x88\xab9-[\x9e|\xf1\xcd")
 
+    def test_ctr_decrypt(self):
+        data = bytes_to_intlist(
+            b"\x03\xc7\xdd\xd4\x8e\xb3\xbc\x1a*O\xdc1\x12+8Aio\xd1z\xb5#\xaf\x08"
+        )
```

Comment:
```suggestion
    def test_ctr_decrypt(self):
        data = bytes_to_intlist(
            b'\x03\xc7\xdd\xd4\x8e\xb3\xbc\x1a*O\xdc1\x12+8Aio\xd1z\xb5#\xaf\x08')
```
single quotes and parens at end of line. same in other places

Reply:
The whole module uses double quotes, but I do agree with that. I correct them all in one.

 I'm not sure on parenthesis, it looks like both acceptable according to PEP8. Personally I don't mind it to be in a new line, but I can see as well, how it might appear to be an eyesore.

Reply:
> I'm not sure on parenthesis, it looks like both acceptable according to PEP8. 

It's best to stick to either one or the other and "same line" is youtube-dl's convention